src/containers/user-profile/comments-with-rating-block/CommentsWithRatingBlock.tsx

```diff
@@ -55,6 +56,28 @@ const CommentsWithRatingBlock = ({
     </MenuItem>
   ))
 
+  const filteredItems = items.filter((item) => {
+    if (filter !== null) {
+      return item.rating === filter
+    }
+    return true
+  })
+
+  const sortedItems = filteredItems.sort((a, b) => {
+    switch (sortBy) {
+      case SortByEnum.Newest:
+        return new Date().getTime() - new Date().getTime()
```

Comment:
You are comparing two Dates that have no relations to user reviews

Reply:
already fixed